<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gerer_parking</class>
 <widget class="QDialog" name="gerer_parking">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>651</width>
     <height>431</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Gestion des Parkings</string>
    </attribute>
    <widget class="QTableView" name="tabParking">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>621</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="idPar">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>300</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="capacitePar">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>300</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="ajouterPar">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>300</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QComboBox" name="etatComboPar">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>300</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
    </widget>
    <widget class="Line" name="line_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>20</width>
       <height>51</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_6">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>10</y>
       <width>20</width>
       <height>51</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_7">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>300</y>
       <width>20</width>
       <height>91</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_8">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>290</y>
       <width>20</width>
       <height>111</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_9">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>290</y>
       <width>20</width>
       <height>111</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>330</y>
       <width>481</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QComboBox" name="idSupprComboPar">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>310</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="supprPar">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>350</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
    <widget class="Line" name="line_11">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>380</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="idModifPar">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>360</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="capaciteModifPar">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>360</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="etatModifComboPar">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>360</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="modifPar">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>360</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Modifier</string>
     </property>
    </widget>
    <widget class="Line" name="line_17">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>350</y>
       <width>20</width>
       <height>51</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_18">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>481</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_19">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>390</y>
       <width>481</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_21">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>481</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_20">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>290</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_22">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>621</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_23">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>621</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QComboBox" name="idModifComboPar">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>360</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="Line" name="line_62">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>20</width>
       <height>51</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Trier par</string>
     </property>
    </widget>
    <widget class="QPushButton" name="rafraichirPar">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Actualiser</string>
     </property>
    </widget>
    <widget class="Line" name="line_61">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>10</y>
       <width>20</width>
       <height>51</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="rechTextPar">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>20</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="triComboPar">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Identifiant</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Capacite</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Etat</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="rechComboPar">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>20</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Identifiant</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Capacite</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Etat</string>
      </property>
     </item>
    </widget>
    <widget class="Line" name="line_59">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>10</y>
       <width>20</width>
       <height>51</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>20</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Rechercher</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Gestion des Places</string>
    </attribute>
    <widget class="QComboBox" name="idSupprComboPl">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>310</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="idModifPl">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>360</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="Line" name="line_41">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>10</y>
       <width>20</width>
       <height>51</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Trier par</string>
     </property>
    </widget>
    <widget class="Line" name="line_42">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>621</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QComboBox" name="etatModifComboPl">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>360</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
    </widget>
    <widget class="Line" name="line_43">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>300</y>
       <width>20</width>
       <height>91</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QComboBox" name="triComboPl">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Identifiant</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Etat</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ID Parking</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>20</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Rechercher</string>
     </property>
    </widget>
    <widget class="QComboBox" name="etatComboPl">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>300</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="idPl">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>300</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="Line" name="line_44">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>290</y>
       <width>20</width>
       <height>111</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="rechTextPl">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>20</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="idModifComboPl">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>360</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="Line" name="line_45">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>290</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="ajouterPl">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>300</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="Line" name="line_46">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>20</width>
       <height>51</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_47">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>481</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_48">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>10</y>
       <width>20</width>
       <height>51</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_49">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>380</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_50">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>390</y>
       <width>481</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_51">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>330</y>
       <width>481</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_52">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>290</y>
       <width>20</width>
       <height>111</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_53">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>621</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QComboBox" name="rechComboPl">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>20</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Identifiant</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Etat</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ID Parking</string>
      </property>
     </item>
    </widget>
    <widget class="Line" name="line_54">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>481</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QTableView" name="tabPlace">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>621</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="supprPl">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>350</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="modifPl">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>360</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Modifier</string>
     </property>
    </widget>
    <widget class="Line" name="line_55">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>10</y>
       <width>20</width>
       <height>51</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_56">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>350</y>
       <width>20</width>
       <height>51</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_58">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>20</width>
       <height>51</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="rafraichirPl">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Actualiser</string>
     </property>
    </widget>
    <widget class="QComboBox" name="idparkingModifComboPl">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>360</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="idparkingComboPl">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>300</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Statistiques</string>
    </attribute>
    <widget class="QChartView" name="statBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>641</width>
       <height>381</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="statCombo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Parkings les plus occupés</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Parkings les moins occupés</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Alerter une panne</string>
    </attribute>
    <widget class="QPushButton" name="envoyer">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>110</y>
       <width>321</width>
       <height>151</height>
      </rect>
     </property>
     <property name="text">
      <string>Envoyer</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QtCharts</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
